.github/workflows/broken-links.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         use-verbose-mode: 'yes'
         config-file: '.github/workflows/markdown.links.config.json'
     - uses: slackapi/slack-github-action@v1.16.0
-      if: always()
+      if: failure()
```

Comment:
Isn't `if: failure()` the default behaviour?

Reply:
Here it says `success()` is the default condition. https://docs.github.com/en/actions/learn-github-actions/expressions#failure
> You can use the following status check functions as expressions in if conditionals. A default status check of success() is applied unless you include one of these functions.

Reply:
Right I got it inverted.